libs/components/src/lib/AboutMe/index.module.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../../../theme/index";
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```

Comment:
please use space from theme instead of this

Reply:
ok, it's done, spacing in theme has an issue, we talking about that in meeting.